CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### [v5.6.2 _(Dec 7, 2023)_](https://github.com/omise/omise-woocommerce/releases/tag/v5.6.2)
```

Comment:
@aashishgurung @muthuswamyopn Can we fix the header levels so that h2 follows from h1?

> In HTML, this would be done using the HTML heading elements (h1, h2, h3, h4, h5, and h6). These allow user agents to automatically identify section headings. Other technologies use other techniques for identifying headers. To facilitate navigation and understanding of overall document structure, authors should use headings that are properly nested (e.g., h1 followed by h2, h2 followed by h2 or h3, h3 followed by h3 or h4, etc.).


https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G141

Reply:
Done.